### PR TITLE
feat: decide between DM or docs when using help command

### PR DESCRIPTION
**Pull request by nistei**
feat: decide between DM or docs when using help command

## Description
Presents an emote selector when using the `.help` command

## Test Results
`.help`

image
`Select 2`

image

## Discord Username
nistei#1362

### CODE REVIEW

File: src/commands/utils/help.ts [hunk 1]
```diff
@@ -9,22 +9,81 @@ export const help: CommandDefinition = {
...
-    executor: (msg) => msg.author.createDM()
-        .then(async (dmChannel) => {
+    executor: async (msg) => {
+        const { author } = msg;
+
+        const embed = makeEmbed({
+            title: 'FlyByWire Simulations | Help',
+            description: makeLines([
+                'Would you like to:',
+                '1. Receive a help DM',
```
Comment (benw202) [inline src/commands/utils/help.ts:19]:
```suggestion
                '1. Receive a list of the available commands as a DM',
```

File: src/commands/utils/help.ts [hunk 2]
```diff
@@ -9,22 +9,81 @@ export const help: CommandDefinition = {
...
+                '2. Receive a link to our docs site with changelogs',
```
Comment (benw202) [inline src/commands/utils/help.ts:20]:
```suggestion
                '2. Receive a link with the available commands/changelog on our docs site ',
```

File: src/commands/utils/help.ts [hunk 3]
```diff
@@ -9,22 +9,81 @@ export const help: CommandDefinition = {
...
+                            } catch (e) {
+                                Logger.error(e);
+                            }
+                        });
+
+                    break;
+                case '2️⃣':
+                    const embed = makeEmbed({
+                        title: 'FlyByWire Simulations | Documentation',
+                        description: 'https://docs.flybywiresim.com/',
```
Comment (benw202) [inline src/commands/utils/help.ts:66]:
```suggestion
                        description: 'https://docs.flybywiresim.com/dev-corner/development-projects/discord-bot/',
```

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/utils/help.ts (hunk 2 above)
Comment (nistei) [reply src/commands/utils/help.ts:20]: This doesn't seem correct.

Comment (benw202) [reply src/commands/utils/help.ts:20]: @nistei My bad! You are correct.

I have edited the suggestion. Feel free to take you own spin on it. I was asked to stress that the docs site has the commands list AND changelog.

**Review by benw202 — APPROVED** [review]

**Comment by benw202** [comment]
LGTM!

Thanks for the work :)

**Pull request merged**